scipy/stats/_levy_stable/__init__.py

```diff
@@ -616,13 +623,14 @@ def phi_5_1(beta, alpha):
     nu_beta = (p95 + p05 - 2 * p50) / (p95 - p05)
 
     if nu_alpha >= 2.439:
-        alpha = np.clip(psi_1_1(nu_beta, nu_alpha)[0], np.finfo(float).eps, 2.)
-        beta = np.clip(psi_2_1(nu_beta, nu_alpha)[0], -1.0, 1.0)
+        eps = np.finfo(float).eps
+        alpha = np.clip(psi_1_1(nu_beta, nu_alpha)[0, 0], eps, 2.)
```

Comment:
I guess `RectBivariateSpline` alows returns 2D output because it treats `x` and `y` as though they were each `1d` arrays, and it always interpolates on a full grid defined by the cartesian product of `x` and `y` (rather than, e.g. `zip`ing them)? 
Coming from `stats`, I would have expected the output to be 0D if `x` and `y` are 0D, 1D if one (or both) of `x` and `y` were `1D`, and 2D if either of `x` or `y` were 2D. That is, the output shape would be the shape you'd get broadcasting `x` against `y`.

Reply:
Here's an analysis by Warren: https://github.com/scipy/scipy/issues/16396#issuecomment-1154175268. My take is that it's a mess :-).

Reply:
OK. I guess my thought is that it would be nice if the recommended interface were to follow the usual broadcasting rules if we're going to eliminate `interp2d`. But it's certainly better than it could be (e.g. always squeezing).